### PR TITLE
List most recently used resource templates

### LINKED ISSUE #1626: List most recently used resource templates
This should be based on the resource templates that the user opens within that session.

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes #1626 

Screen Shot 2019-11-07 at 9 27 23 AM

### CODE REVIEW

File: __tests__/actionCreators/resourceTemplates.test.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ describe('fetchResourceTemplate', () => {
       expect(resourceTemplate).toEqual(templateResponse.response.body)
       expect(store.getActions()).toEqual([
         { type: 'RESOURCE_TEMPLATE_LOADED', payload: templateResponse.response.body },
+        { type: 'ADD_TEMPLATE_HISTORY', payload: templateResponse.response.body },
```
Comment (justinlittman) [inline __tests__/actionCreators/resourceTemplates.test.js]: Ignore me.

**Comment by jcoyne** [comment]
Agree about not showing the section, but "most frequently" is the text on the design comps as is the position of the search box.

**Comment by jcoyne** [comment]
@astridu can you tell us how you would have us proceed?

**Comment by astridu** [comment]
It was decided to use "Most recently used" after I created the mockup.

**Comment by astridu** [comment]
Most recently used section should be collapsible. When users comes to the screen, the section is collapsed by default.

**Pull request merged**